ninetofiver/models.py

```diff
@@ -371,6 +371,7 @@ class UserInfo(BaseModel):
     gender = models.CharField(max_length=2, choices=GENDER)
     country = CountryField()
     join_date = models.DateField(_("Became Inuit on"), default=date.today)
+    redmine_user_id = models.CharField(max_length=255, blank=True, null=True)
```

Comment:
Decouple redmine specifics from 925r core models. Create a "RedmineUserInfo" model which can OPTIONALLY be linked to Users.